tests/robottelo/test_cli.py

```diff
@@ -334,13 +332,13 @@ def test_info_parsing_response(self, construct, execute, parse):
         )
         parse.called_once_with('some_response')
 
-    @mock.patch('robottelo.cli.base.Base.command_requires_org')
-    def test_list_requires_organization_id(self, _):
-        """Check list raises CLIError with organization-id is not present in
-        options
-        """
-        with pytest.raises(CLIError):
-            Base.list()
+    # @mock.patch('robottelo.cli.base.Base.command_requires_org')
+    # def test_list_requires_organization_id(self, _):
+    #     """Check list raises CLIError with organization-id is not present in
+    #     options
+    #     """
+    #     with pytest.raises(CLIError):
+    #         Base.list()
```

Comment:
I think this is connected to what jake mentioned in the code comments before. So just mentioning it so he doesn't forget to change it also ;)